.github/CONTRIBUTING.md

```diff
@@ -94,3 +104,17 @@ There are two ways of running the `goat` locally:
 
 1. If adding linters to `linters.json`, the `executor` is an optional member of the linter object.
 2. The `autofix` member is also optional. If a linter has an autofix command-line option, this field holds those arguments.
+
+### Iterating on Taskfile.yml files
+
+Make sure you:
+
+1. Update the submodule to point to your branch, otherwise your changes under `Task/` will not be effective for the parent `Taskfile.yml`. We can't avoid this
+   because the `Task/**/Taskfile.yml` files have a relative `dir:` and if the `goat` uses it directly (avoiding a submodule) then the relative dir is one folder
+   level different than every project that uses it. Consider a helper alias like this for use when developing:
+
+   ```bash
+   alias updategoat='git add -A && git commit -m "Update goat" && ggp && cd goat && ggpull && cd .. && git add -A && git commit -m "Update submodule" && ggp
+   ```
+
+1. Repoint the submodule back to `main` after merging your PR
```

Comment:
TODO: Automate this.  Repoint back to main and push that as a part of `task release`?

Reply:
Added this to `task release`